src/document.c

```diff
@@ -837,11 +826,11 @@ GeanyDocument *document_new_file(const gchar *utf8_filename, GeanyFiletype *ft,
 	sci_set_undo_collection(doc->editor->sci, FALSE); /* avoid creation of an undo action */
 	if (text)
 	{
-		GString *template = g_string_new(text);
-		utils_ensure_same_eol_characters(template, file_prefs.default_eol_character);
+		GString *text_wr = g_string_new(text);
```

Comment:
i guess "template" naming is an artifact of times when this function was used only to create files from template, now it is also used for other purposes